ci/cron/daily-compat.yml

```diff
@@ -196,3 +196,61 @@ jobs:
         env:
           GCRED: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
       - template: ../daily_tell_slack.yml
+
+  - job: blackduck_scan
+    timeoutInMinutes: 1200
+    pool:
+      name: linux-pool
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)        
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: ./build.sh "_$(uname)"
```

Comment:
`./build.sh` also runs tests. Do we need this? If not, `bazel build //....` will be significantly faster.

Reply:
changed